### PR TITLE
[testing-library__jest-dom] Fix hasClass types

### PR DESCRIPTION
**Pull request by GusevDV**
According to the documentation the hasClass method supports two params. In index.d.ts declares only one. I've added the second param. 

Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test YOUR_PACKAGE_NAME`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <<https://github.com/testing-library/jest-dom/tree/v5.9.0#tohaveclass>>
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.
- [x] Include [tests for your changes](https://github.com/DefinitelyTyped/DefinitelyTyped#testing)
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`. If for reason the any rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]` and not for whole package so that the need for disabling can be reviewed.

### CODE REVIEW

File: types/testing-library__jest-dom/index.d.ts [hunk 1]
```diff
@@ -259,11 +259,14 @@ declare namespace jest {
...
          * expect(deleteButton).toHaveClass('btn')
          * expect(deleteButton).toHaveClass('btn-danger xs')
+         * expect(deleteButton).toHaveClass('btn xs btn-danger', {exact: true})
+         * expect(deleteButton).not.toHaveClass('btn xs btn-danger', {exact: true})
          * expect(noClasses).not.toHaveClass()
          * @see
          * [testing-library/jest-dom#tohaveclass](https:github.com/testing-library/jest-dom#tohaveclass)
          */
         toHaveClass(...classNames: string[]): R;
+        toHaveClass(classNames: string,  options?: {exact: boolean}): R;
```
Comment (jgoz) [inline types/testing-library__jest-dom/index.d.ts:269]:
```suggestion
        toHaveClass(classNames: string, options?: { exact: boolean }): R;
```

**Review by jgoz — CHANGES_REQUESTED** [review]
Thank you, just a formatting suggestion for consistency with the rest of the file.

**Review by jgoz — APPROVED** [review]

**Comment by GusevDV** [comment]
@jgoz Thank you for review. I've changed the testing-library__jest-dom-global-tests.ts too, because I've found problem with formatting. Can you make review again?

**Review by jgoz — APPROVED** [review]
Looks good, thanks @GusevDV

**Comment by GusevDV** [comment]
Ready to merge

**Pull request merged**